### ISSUE TITLE
Impossible use the char "~" in defold editor

### ISSUE THREAD

**Issue by cpusam** [defold/editor2-issues#2646]
### Expected behaviour
Just put a single '~' using my abnt (brasilian keyboard) in the editor
### Actual behaviour
not put the '~' when needs to use the operator ~=
### Steps to reproduce
I think for this it's needed an abnt keyboard or change the main layout of the OS
<hr/>

<table>
<tr><td>Build time</td><td>2019-05-28T17:15:49.287628</td></tr>
<tr><td>Defold channel</td><td>editor-alpha</td></tr>
<tr><td>Defold editor sha</td><td>d23897acd4d7643bc1293097e17a779201e58d6b</td></tr>
<tr><td>Defold engine sha</td><td>838cecd7a26c932e6be73421d98e51ba12f1d462</td></tr>
<tr><td>Defold version</td><td>1.2.155</td></tr>
<tr><td>GPU</td><td>llvmpipe (LLVM 7.0, 256 bits)</td></tr>
<tr><td>GPU Driver</td><td>3.1 Mesa 18.2.8</td></tr>
<tr><td>Java version</td><td>11.0.1+13</td></tr>
<tr><td>OS arch</td><td>amd64</td></tr>
<tr><td>OS name</td><td>Linux</td></tr>
<tr><td>OS version</td><td>4.15.0-51-generic</td></tr>
<table>

**Comment by matgis-king** [comment]
The behavior of the <kbd>&tilde;</kbd> key seems to differ on non-US keyboard layouts. It will not appear immediately when typed. It awaits a second keypress so that the `~` can be composited onto another character. I.e. pressing <kbd>&tilde;</kbd><kbd>o</kbd> results in a `õ`. To get a tilde by itself you have to press <kbd>&tilde;</kbd><kbd>space</kbd>.

We tried this on both macOS and Windows, and it seems to work as described above on both platforms. Are you saying <kbd>&tilde;</kbd><kbd>space</kbd> does not produce a `~` on your machine?

**Comment by cpusam** [comment]
> We tried this on both macOS and Windows, and it seems to work as described above on both platforms. Are you saying ˜space does not produce a `~` on your machine?

No, just pressing ~ (or hold it) don't produces any char, nothing happens.
Pressing  ~ + 'space'  produces a char equals to a single yellow point, not the point made with '.', but a point that can't be copying into the clipboard.
Pressing ~ + o results in just 'o' and not 'õ'. It is like the key '~' doesn't exists
In all this cases, the key ~ appears like it not exists, but other keys are ok.

Maybe could be my system, that isn't the Ubuntu, but the XUbuntu 18, any difference in this?

**Comment by matgis-king** [comment]
Thanks for the additional info. I've been able to reproduce the issue Using a Swedish keyboard on Ubuntu 18.04 running under Parallels, so there's definitely a problem.
